Anlab.Mvc/Controllers/AccountController.cs

```diff
@@ -90,8 +86,123 @@ public class AccountController : Controller
             return View(model);
         }
 
-        //
-        // GET: /Account/Register
+        [HttpGet]
+        [AllowAnonymous]
+        public async Task<IActionResult> LoginWith2fa(bool rememberMe, string returnUrl = null)
+        {
+            // Ensure the user has gone through the username & password screen first
+            var user = await _signInManager.GetTwoFactorAuthenticationUserAsync();
+
+            if (user == null)
+            {
+                throw new ApplicationException($"Unable to load two-factor authentication user.");
```

Comment:
Unnecessary $

Reply:
@jpknoll yea, don't know why the template does that https://github.com/aspnet/Templates/blob/dev/template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/StarterWeb-CSharp/Controllers/AccountController.cs#L150